dev/CommonStyles/MediaTransportControls_themeresources.xaml

```diff
@@ -11,24 +11,24 @@
             <StaticResource x:Key="MediaTransportControlsPanelBackground" ResourceKey="GhostFillColorSecondaryBrush" />
             <StaticResource x:Key="MediaTransportControlsFlyoutBackground" ResourceKey="SystemControlTransientBackgroundBrush" />
             <StaticResource x:Key="MediaTransportControlsThumbBorderBackground" ResourceKey="ControlAAFillColorDefaultBrush" />
-            <StaticResource x:Key="MediaTransportControlsTimeElapsedText" ResourceKey="TextFillColorSecondaryBrush" />
-            <StaticResource x:Key="MediaTransportControlsMediaText" ResourceKey="TextFillColorPrimaryBrush" />
+            <StaticResource x:Key="MediaTransportControlsFillTimeElapsedText" ResourceKey="TextFillColorSecondaryBrush" />
```

Comment:
We can't just rename resources like this, because there might be an app which references the resource.  We should put a resource with the old name and value in the dictionary as well (or a separate dictionary for deprecated resources) and put a comment saying that it is no longer used.

Reply:
I don't think it is necessary since this change only resides in **token-experiment** branch (it's not in master) and was introduced in this [commit](https://github.com/microsoft/microsoft-ui-xaml/pull/3728/files). Unless a developer cloned the token-experiment branch after Dec 1 and created an app what is very unlikely :)

Reply:
These are new resources so we should be okay to rename them.

Reply:
sweet